src/connectors/sso.ts

```diff
@@ -5,7 +5,7 @@ document.addEventListener('DOMContentLoaded', (event) => {
     const code = getQsParam('code');
     const state = getQsParam('state');
 
-    if (state != null && state.endsWith(':clientId=browser')) {
+    if (state != null && state.endsWith(':clientId=browser_identifier=sso')) {
```

Comment:
```suggestion
    if (state != null && state.includes(':clientId=browser')) {
```

Organization identifier is a user-set value, so we can't use it explicitly in an endsWith. I'd just check the clientId parameter standalone with an includes.

Reply:
ah good point! that's what happens when you tunnel vision your test case.